ftw/shop/tests/test_cart.py

```diff
@@ -190,7 +191,7 @@ def test_checkout(self):
         cart.checkout()
 
         order = omanager.getOrders()[0]
-        self.assertEquals(order.total, Decimal('7.15'))
+        self.assertEquals(order.total, Decimal('14.30'))
```

Comment:
You just extend existing tests, but you added a new feature. The current testcases are very large, but perhaps you can add own testscases where you just test your feature here.
(this comment is for all the tests added in this commit)